.github/workflows/build.yml

```diff
@@ -67,10 +68,10 @@ jobs:
           cache: 'maven'
 
       - name: Build with Maven
-        run: mvn -B clean install -Dno-format
+        run: mvn -B clean install -Dno-format -DskipTests
 
       - name: Build with Maven (Native)
-        run: mvn -B install -Dnative -Dquarkus.native.container-build -Dnative.surefire.skip
+        run: mvn -B install -Dnative -Dquarkus.native.container-build -Dnative.surefire.skip -DskipTests
```

Comment:
Let's run with tests instead of skipping tests in order to capture any issues in native mode.

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/pull/218/commits/a04324ddaa3f3f8d3a13bff17b498d147d3ff4ac.